templates/rke2-airgap/overlay/opt/corral/rke2/init-agent.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+
+mkdir -p /etc/rancher/rke2
+cat > /etc/rancher/rke2/config.yaml <<- EOF
+server: https://${CORRAL_server_ip}:9345
+system-default-registry: ${CORRAL_registry_fqdn}
+token: ${CORRAL_node_token}
+EOF
+
+INSTALL_RKE2_ARTIFACT_PATH=/root/rke2-artifacts INSTALL_RKE2_TYPE=agent sh /root/rke2-artifacts/install.sh 
```

Comment:
Do we need a new package for this logic, or can we update this the existing logic:
```
curl -sfL https://get.rke2.io | INSTALL_RKE2_VERSION=${CORRAL_kubernetes_version} sh -
```
to get something like 
```
echo "$($CORRAL_rke2_install_command)" | INSTALL_RKE2_VERSION="${CORRAL_kubernetes_version}" INSTALL_RKE2_TYPE="agent" "$EXTRA_INSTALL_ARGS" sh - 
```

It seems like we might want to add that configuration directly into the scripts, given maintaining two separate sets of install scripts for rke2 and k3s each could cause a little extra development churn. Thoughts?

Reply:
I think this could work, I'll try that out.